policies/coding-style.md

```diff
@@ -646,10 +646,13 @@ or have important caveats:
   on some platforms.
 
 Existing public APIs using deprecated types must be maintained, but new APIs and
-new code should avoid use of them.
+new code should avoid use of them. Deprecation does not mean that existing code
+need be converted when it is modified, and other code may still use deprecated
```

Comment:
Typo: needs

Reply:
Either wording is OK with me, but this is grammatical.